src/main/app/claims/claimStoreClient.ts

```diff
@@ -215,4 +217,17 @@ export class ClaimStoreClient {
         }
       })
   }
+
+  saveClaimantResponse (claim: Claim, draft: Draft<DraftClaimantResponse>, user: User): Promise<void> {
+    const response = ClaimantResponseConverter.covertToClaimantRespose(draft.document)
```

Comment:
Please remove from the client. It is not client's responsibility to convert from draft.

Reply:
Can you please let me know where it should be. FYI...I see other methods like saveClaim, saveResponseForUser where it has been done in a similar fashion.

Reply:
Calling converters in clients classes was never right / desired thing.

Reply:
Just call converter to prepare payload in place you call the client, most likely check-and-send route.